openff/toolkit/utils/nagl_wrapper.py

```diff
@@ -17,6 +17,12 @@
 
 
 class NAGLToolkitWrapper(ToolkitWrapper):
+    """NAGL toolkit wrapper for applying partial charges with a GCN model.
```

Comment:
Re-surfacing my suggestion for "OpenFF NAGL toolkit wrapper for ... " which you are welcome to pass on